src/components/LoanComponents/APR.tsx

```diff
@@ -0,0 +1,19 @@
+import React,{useContext, useEffect} from "react";
+import { AppContext } from "../../ValueProvider";
+
+export default function APRfield() {

```

Comment:
For the future, make sure your components have the same name as the default export